### PR TITLE
Update bower build process

### PR DESCRIPTION
**Pull request by dkniffin**
Bower wasn't working, because the dist files weren't being pushed, and they weren't defined in the bower.json. This should fix it.

### CODE REVIEW

**Comment by kekscom** [comment]
How can this place the dist files? Can it?

File: build/dist.sh [hunk 1]
```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+set -x
+set -e # exit with nonzero exit code if anything fails
+
+git config user.name "Travis CI"
+git add --force dist/OSMBuildings*
```
Comment (dkniffin) [inline build/dist.sh]: This should be adding the dist files, but it appears to not be working.

**Comment by kekscom** [comment]
What, if we un-ignore them?

- a basic build in any branch should not touch those
- on master, they would be build upon push/merge
- such build should not affect a branches unless explicitly merged

@rbuch703 Am I missing something?

**Comment by rbuch703** [comment]
Could it be that the scripts are broken? The `deploy.sh` calls `./dist.sh`, which implies that the folder that contains `dist.sh` is the current working directory - meaning `build`. But there shouldn't be a `dist/OSMBuildings` folder in `build` to add to the repo.

**Comment by dkniffin** [comment]
Found it. There's a couple problems. The first is [this](https://travis-ci.org/OSMBuildings/OSMBuildings/builds/124018975#L312). The second is what @rbuch703 said. I'll fix these this morning, probably.

**Pull request merged**